resources/multisite_drupal_standard.make

```diff
@@ -212,6 +212,8 @@ projects[entity_translation][patch][] = https://www.drupal.org/files/issues/enti
 projects[entity_translation][patch][] = https://www.drupal.org/files/issues/entity_translation-pathauto_update-2743685-2_0.patch
 ; https://www.drupal.org/node/2877074
 projects[entity_translation][patch][] = https://www.drupal.org/files/issues/entity_translation-fix_content_translation_test-2877074-4.patch
+; Issue MULTISITE-15175 : Undefined index path wildcard
+projects[entity_translation][patch][] = patches/entity_translation-fix-for-wildcard.patch
```

Comment:
This patch is on d.org, it is better to use it.
https://www.drupal.org/node/2840645

Reply:
oh right ! and this patch was posted by us ^^

Fixed